src/main/java/io/jsonwebtoken/IncorrectClaimException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2015 jsonwebtoken.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.jsonwebtoken;
+
+/**
+ *  IncorrectClaimException is a subclass of the {@link InvalidClaimException} that is thrown after it is found that an
```

Comment:
That this is a subclass is obvious just by the nature of the generated JavaDoc and a little heavy-handed for documentation IMO.  I recommend keeping it direct and simple:

> Exception thrown when parsing a required claim and the value does not equal the required value.